tools/specs/lib/gzipTilesetSpec.js

```diff
@@ -173,21 +172,20 @@ describe('gzipTileset', function() {
     });
 
     it('accepts custom writeCallback that returns a promise', function (done) {
-        var outputDirectory = gzippedDirectory;
         var writeCallback = function(file, data) {
-            var outputFile = path.join(outputDirectory, file);
-            return fsExtraOutputFile(outputFile, data);
+            return fsExtraOutputFile(file, data);
         };
         var gzipOptions = {
             inputDirectory : tilesetDirectory,
+            outputDirectory : gzippedDirectory,
```

Comment:
The idea behind custom write callbacks was that you didn't need to supply an output directory to the stage because that information only needs to be known by the callback. I think the old way with using a closure for `getDefaultWriteCallback` was more flexible.